examples/externalProfileDemo.html

```diff
@@ -0,0 +1,337 @@
+                document
+                    .querySelector('.manual-example #contactButton')
+                    .addEventListener('click', function () {
+                        let contactId = document.querySelector('#contact-id').value.trim();
+                        if (contactId) {
+                            clientApp.contacts.showExternalContactProfile(contactId);
+                        }
+                    });
+                document
+                    .querySelector('.manual-example #orgButton')
+                    .addEventListener('click', function () {
+                        let externalOrganizationId = document.querySelector('#org-id').value.trim();
+                        if (externalOrganizationId) {
+                            clientApp.contacts.showExternalOrganizationProfile(externalOrganizationId);
+                        }
+                    });
+
+                let redirectUrl = window.location.origin;
+                if (!redirectUrl) {
+                    redirectUrl = window.location.protocol + '//' + window.location.host;
+                }
+                redirectUrl += window.location.pathname;
+
+                // Authenticate with PureCloud
+                let authenticated = false;
+                let userDataAcquired = false;
+
+                client
+                    .loginImplicitGrant(clientId, redirectUrl, {
+                        state: 'pcEnvironment=' + pcEnvironment
+                    })
+                    .then(function () {
+                        updateProgressBar(100);
+                        authenticated = true;
+                        return new platformClient.UsersApi().getUsersMe({
+                            expand: ['authorization']
+                        });
+                    })
+                    .then(function (profileData) {
+                        userDataAcquired = true;
+                        // Check if the current user will be able to view external contacts
+                        let permissions = profileData.authorization.permissions;
+                        if (checkPermission(permissions, 'externalContacts:contact:view')) {
+                            requestExternalContacts().then(data => {
+                                let entities = data.entities;
+                                // Build table with data from api call
+                                for (let i = 0; i < entities.length; i++) {
+                                    document.getElementById('tableBody').insertAdjacentHTML('beforeend',
+                                        `<tr>
+                                            <td>
+                                                <a id="entity-${i}">${entities[i].lastName}, ${entities[i].firstName}</a>
+                                            </td>
+                                            <td>
+                                                <a id="org-${i}">${entities[i].externalOrganization.id}</a>
```

Comment:
This is failing in my org.  Seems like I have a contact without an org.  So, you'll probably need to check for that before adding this table cell.